packages/image-block/src/components/DownloadAndAttachments.tsx

```diff
@@ -35,9 +35,12 @@ export const DownloadAndAttachments = ({
                 data-test-id="buttons-wrapper"
                 style={getTotalImagePadding(blockSettings)}
             >
-                {image && isDownloadable(blockSettings.security, blockSettings.downloadable, assetDownloadEnabled) && (
-                    <DownloadButton onDownload={() => appBridge.dispatch({ name: 'downloadAsset', payload: image })} />
-                )}
+                {image?.isDownloadProtected === false &&
```

Comment:
Is there a scenario where `image?.isDownloadProtected` is undefined?

Reply:
If I adjust it, we would get a typecheck error on the `dispatch` of the `download`